web/ui/react-app/src/pages/targets/ScrapePoolList2.tsx

```diff
@@ -0,0 +1,131 @@
+import { KVSearch } from '@nexucis/kvsearch';
+import { usePathPrefix } from '../../contexts/PathPrefixContext';
+import { useFetch } from '../../hooks/useFetch';
+import { API_PATH } from '../../constants/constants';
+import { groupTargets, ScrapePool, ScrapePools, Target } from './target';
+import { withStatusIndicator } from '../../components/withStatusIndicator';
+import React, { ChangeEvent, useState } from 'react';
+import { Collapse, Container, Input, InputGroup, InputGroupText } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { ScrapePoolContent } from './ScrapePoolContent';
+import Filter, { Expanded, FilterData } from './Filter';
+import { useLocalStorage } from '../../hooks/useLocalStorage';
+import styles from './ScrapePoolPanel.module.css';
+import { ToggleMoreLess } from '../../components/ToggleMoreLess';
+
+interface ScrapePoolListProps {
+  activeTargets: Target[];
+}
+
+const kvSearch = new KVSearch({
+  shouldSort: true,
+  indexedKeys: ['labels', 'scrapePool', ['labels', /.*/]],
+});
+
+interface PanelProps {
+  scrapePool: string;
+  targetGroup: ScrapePool;
+  expanded: boolean;
+  toggleExpanded: () => void;
+}
+
+function ScrapePoolPanel(props: PanelProps): JSX.Element {
```

Comment:
Could you keep this in a separate file as it was before? We always put components in their own files unless one is really just a totally dumb wrapper around another.

I'd also remove the old implementations and just replace them with the new ones vs. having new ones with a `2` appended.

Reply:
in this case, I think it's a quite dumb wrapper. But if you really want I can move it in a different file of course.